tf-modules/cumulus/variables.tf

```diff
@@ -182,7 +182,7 @@ variable "bucket_map_key" {
 variable "cmr_custom_host" {
   description = "Custom host to use for CMR requests"
   type        = string
-  default     = ""
+  default     = null
```

Comment:
This variable is used to set various Lambda environment variables. And it turns out that in Terraform, if the variable used to set an environment variable is `null`, it just excludes it altogether, which is preferable to including and setting it to an empty string